recipes/image_classification/classification.py

```diff
@@ -867,6 +867,41 @@
 )
 
 
+def optimized_params(args, phinet_params, num_classes, save_path):
```

Comment:
wouldn't it be better to pass args directly from NAS like this? or at least merge multiple dictionaries, one for search and one for data handling?
The search space output should be a dictionary passed from the NAS output, the other is a "context" dictionary that depends on the experimental setup you are using